### PR TITLE
implement the blinding key derivation as per Trezor

### PR DESCRIPTION
**Pull request by glslang**

### CODE REVIEW

File: src/signer.cpp [hunk 1]
```diff
@@ -165,6 +176,38 @@ namespace sdk {
...
+    {
+        GDK_RUNTIME_ASSERT(m_net_params.liquid());
+
+        const std::string domain_str{ "Symmetric key seed" };
+        const auto domain = ustring_span(domain_str);
+        auto root = hmac_sha512(domain, seed);
+
+        const std::string label_str{ "SLIP-0077" };
+        const auto label = ustring_span(label_str);
+        auto node = hmac_sha512(gsl::make_span(root).first(EC_PRIVATE_KEY_LEN), label);
```
Comment (romanz) [inline src/signer.cpp]: `label` should be prefixed with the `'\x00'` byte before applying HMAC:
https://github.com/satoshilabs/slips/blob/master/slip-0021.md#child-node-derivation
https://github.com/satoshilabs/slips/pull/662/files#diff-a3d7c0a374f7cf006bee833349d8d689R42

**Comment by real-or-random** [comment]
LGTM (except for the bug found by @romanz)

**Comment by real-or-random** [comment]
utACK

**Comment by romanz** [comment]
LGTM 02d11a8

**Comment by glslang** [comment]
moved to wally.

**Pull request closed without merging**